ml/jml/fasttext_classifier.cc

```diff
@@ -0,0 +1,317 @@
+/* fasttext_classifier.cc                                                -*- C++ -*-
+   Mathieu Marquis Bolduc, 27 fevrier 2017
+   This file is part of MLDB. Copyright 2017 mldb.ai inc. All rights reserved.
+*/
+
+#include "fasttext_classifier.h"
+#include "classifier_persist_impl.h"
+#include <boost/progress.hpp>
+#include <boost/timer.hpp>
+#include <functional>
+#include "mldb/jml/utils/vector_utils.h"
+#include "mldb/ml/jml/registry.h"
+
+#include "mldb/base/exc_assert.h"
+#include "mldb/jml/utils/smart_ptr_utils.h"
+
+#include "mldb/ext/fasttext/src/fasttext.h"
+#include "mldb/ext/fasttext/src/args.h"
+#include "mldb/ext/fasttext/src/model.h"
+
+
+using namespace std;
+using namespace ML::DB;
+
+namespace ML {
+
+
+/*****************************************************************************/
+/* DECISION_TREE                                                             */
+/*****************************************************************************/
+
+FastTest_Classifier::FastTest_Classifier()
+{
+
+}
+
+FastTest_Classifier::
+FastTest_Classifier(DB::Store_Reader & store,
+              const std::shared_ptr<const Feature_Space> & fs)    
+{
+    throw Exception("FastTest_Classifier constructor(reconst): not implemented");
+}
+    
+FastTest_Classifier::
+FastTest_Classifier(std::shared_ptr<const Feature_Space> feature_space,
+              const Feature & predicted)   
+ : Classifier_Impl(feature_space, predicted)
+{
+    ExcAssert(Classifier_Impl::feature_space());
+}
+    
+FastTest_Classifier::
+~FastTest_Classifier()
+{
+}
+    
+void
+FastTest_Classifier::
+swap(FastTest_Classifier & other)
+{
+    Classifier_Impl::swap(other);
```

Comment:
Well we're going to need that call regardless when/if we implement it, so its to not forget it :)